### PR TITLE
Adding mentor to FOSSASIA website

### PR DESCRIPTION
**Pull request by Mridulbirla13**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- Updated the mentors.yml file situated in the Data folder.
- Uploaded the image in the name of Pawan.jpg in images/mentors
- Participating in Google Code In 2016
   Here is my task instance link - (https://codein.withgoogle.com/dashboard/task-instances/6250221841416192/)
- Here is my live link - (https://github.com/fossasia/gci16.fossasia.org/pull/849)
- Here is my forked repository - (https://github.com/Mridulbirla13/gci16.fossasia.org)

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -181,6 +181,10 @@
   github: ananyo2012
   image: MaitiAnanyo.jpg
   twitter: MaitiAnanyo
+  
+- name: Pawan Pal
+  github: Pawan Pal
```
Comment (robbyoconnor) [inline _data/mentors.yml]: You didn't even try to find it did you?

**Review by robbyoconnor — CHANGES_REQUESTED** [review]
Try again -- googling his name yield his github. If it wasn't for the blatent lack of effort, I'd do it for you but i'm instead going to make you do it.

File: _data/mentors.yml (hunk 1 above)
Comment (robbyoconnor) [reply _data/mentors.yml]: This is really, really bad.

**Comment by robbyoconnor** [comment]
@I would also like to remind you that putting forth work where you clearly didn't even try -- pretty much means you won't win pretty much definitively.  Please put forth effort. It makes me sad when you don't try.

File: _data/mentors.yml (hunk 1 above)
Comment (robbyoconnor) [reply _data/mentors.yml]: Here's where you're wrong: github usernames do not contain spaces. Find his twitter as well. It's not hard to find. Google it.

Comment (abishekvashok) [reply _data/mentors.yml]: Yep please google atleast and input only correct credentials

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please add correct credentials...

File: _data/mentors.yml (hunk 1 above)
Comment (robbyoconnor) [reply _data/mentors.yml]: Actually I see what happened. The username is highlighted -- it's also in the URL! 

image

**Comment by robbyoconnor** [comment]
# Mentors: do not approve his [task](https://codein.withgoogle.com/dashboard/task-instances/6250221841416192/) with this PR submitted as the work product. Comments have been left on the task. 

@Mridulbirla13, You really didn't put forth any effort -- I'm half considering unassigning this task -- I will merge this but it's not even worth the task at this point. The blatent lack of effort on your part is sad. Did you know his github username is also his twitter handle? I'm not giving you the task. You didn't earn it.

**Pull request merged**